Lib/test/test_typing.py

```diff
@@ -4585,8 +4585,12 @@ def test_typeddict_create_errors(self):
 
         with self.assertRaises(TypeError):
             TypedDict(_typename='Emp', name=str, id=int)
-        with self.assertRaises(TypeError):
-            TypedDict('Emp', _fields={'name': str, 'id': int})
+        
+        # We raise a DeprecationWarning for the keyword syntax
+        # before the TypeError.
+        with self.assertWarns(DeprecationWarning):
+            with self.assertRaises(TypeError):
+                TypedDict('Emp', _fields={'name': str, 'id': int})
```

Comment:
I think we can remove tests with `_fileds`. This field is long gone: https://github.com/python/cpython/commit/8fc5839a9def34c13b6025c291434ba5fb5d6442#diff-ddb987fca5f5df0c9a2f5521ed687919d70bb3d64eaeb8021f98833a2a716887L1723-L1727

Reply:
Agree!

Reply:
Good point, removed it